### PR TITLE
SAK-29372 sakai as LTI provider to trust TC emailaddress as userid

### PR DESCRIPTION
**Pull request by pushyamig**
@bkirschn @dlhaines

### CODE REVIEW

**Comment by pushyamig** [comment]
Done with the Code review Changes.

File: basiclti/basiclti-util/pom.xml [hunk 1]
```diff
@@ -44,6 +44,10 @@
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-databind</artifactId>
 		</dependency>
+		<dependency>
+            <groupId>org.sakaiproject.kernel</groupId>
+            <artifactId>sakai-component-manager</artifactId>
+        </dependency>
         <dependency>
             <groupId>commons-logging</groupId>
             <artifactId>commons-logging</artifactId>
```
Comment (pushyamig) [inline basiclti/basiclti-util/pom.xml]: I see the file is properly indented, it Github that is showing like this

Comment (buckett) [inline basiclti/basiclti-util/pom.xml]: It's because you're indenting some lines with tabs and some with spaces.

Comment (ottenhoff) [inline basiclti/basiclti-util/pom.xml]: Github is accurate. You are most likely using a different combination of spaces and tabs as the previous lines.

Comment (pushyamig) [inline basiclti/basiclti-util/pom.xml]: Fixed it.

**Pull request merged**